hibernate-core/src/main/java/org/hibernate/metamodel/mapping/ColumnConsumer.java

```diff
@@ -15,5 +15,6 @@
 	void accept(
 			String containingTableExpression,
 			String columnExpression,
+			boolean isColumnExpressionFormula,
```

Comment:
Outside the scope of this PR, but as a larger question here we also need to start thinking about how `@ColumnTransformer` is supported here.  May be better to just pass along the "read" and "write" fragment portions?

Reply:
Good point. I am gonna create a Trello ticket for `@ColumnTransformer` soon. Yeah, adding "read" and "write" fragment seems accommodate every possible scenario (normal column, formula and columnTransformer). But from design perspective, maybe it is an idea to create a higher level class to include the three cases (maybe via class hierarchy). That would simplify code understanding and ease code maintaining.